content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -120,19 +120,19 @@ You can modify the default CoreDNS behavior by modifying the ConfigMap.
 CoreDNS has the ability to configure stubdomains and upstream nameservers using the [forward plugin](https://coredns.io/plugins/forward/).
 
 #### Example
-If a cluster operator has a [Consul](https://www.consul.io/) domain server located at 10.150.0.1, and all Consul names have the suffix .consul.local. To configure it in CoreDNS, the cluster administrator creates the following stanza in the CoreDNS ConfigMap.
+If a cluster operator has a [Consul](https://www.consul.io/) domain server located at `10.150.0.1`, and all Consul names have the suffix .consul.local. To configure it in CoreDNS, the cluster administrator creates the following stanza in the CoreDNS ConfigMap.
```

Comment:
```suggestion
If a cluster operator has a [Consul](https://www.consul.io/) domain server located at `10.150.0.1`,
and all Consul names have the suffix .consul.local. To configure it in CoreDNS,
the cluster administrator creates the following stanza in the CoreDNS ConfigMap.
```
It might be better to give some newline chars

Reply:
thank.